tracing-subscriber/src/fmt/format/json.rs

```diff
@@ -236,6 +236,28 @@ where
                 serializer.serialize_entry("target", meta.target())?;
             }
 
+            if self.display_thread_name {
+                match std::thread::current().name() {
+                    Some(name) => {
+                        serializer.serialize_entry("threadName", name)?;
+                    }
+                    None => {
+                        // fall-back to thread id when name is absent and id's are not enabled
+                        if !self.display_thread_id {
+                            serializer.serialize_entry(
+                                "threadId",
```

Comment:
I think that in this case, the key should maybe still be `threadName` --- if you enable the thread name option, you might reasonably expect all the output JSON to have a field named `threadName`, and might write parsing code that expects that...

What do you think?

Reply:
Yep, Makes sense!